### ISSUE TITLE
egui 0.28 support

### ISSUE THREAD

**Issue by wk39** [PingPongun/egui_struct#5]
Hello,

First of all, thank you for the fantastic crate!
I'm a big fan of your work, and it's the reason I prefer using egui over other options.

I'm considering upgrading from egui 0.27 to 0.28, which was released a few days ago.

If you have any plans to update the crate, I'd love to hear about them.

Thank you!

**Comment by PingPongun** [comment]
Hi,
I will try to add egui v0. 28 support tomorrow.
After quick scan through egui changelog, I see there will be few changes required, but nothing complicated I think

**Comment by PingPongun** [comment]
`egui v0.28` is supported in freshly released `egui-struct v0.4.2`.
Usage:
```toml
egui_struct = { version = "0.4.2", default-features = false, features = [ "egui28" ] }
```
